product-cli/cmd/cluster/cluster.go

```diff
@@ -14,25 +14,26 @@ import (
 
 func NewCreateCommands() *cobra.Command {
 	opts := &core.CreateOptions{
-		Namespace:                      "clusters",
-		Name:                           "example",
-		ReleaseImage:                   "",
-		PullSecretFile:                 "",
-		ControlPlaneAvailabilityPolicy: "HighlyAvailable",
-		Render:                         false,
-		InfraID:                        "",
-		ServiceCIDR:                    "172.31.0.0/16",
-		ClusterCIDR:                    "10.132.0.0/14",
-		Wait:                           false,
-		Timeout:                        0,
-		ExternalDNSDomain:              "",
 		AdditionalTrustBundle:          "",
+		Arch:                           "amd64",
+		ClusterCIDR:                    "10.132.0.0/14",
+		ControlPlaneAvailabilityPolicy: "HighlyAvailable",
 		ImageContentSources:            "",
-		NodeSelector:                   nil,
+		InfraID:                        "",
+		InfrastructureJSON:             "",
 		Log:                            log.Log,
+		Name:                           "example",
+		Namespace:                      "clusters",
 		NodeDrainTimeout:               0,
+		NodeSelector:                   nil,
 		NodeUpgradeType:                v1beta1.UpgradeTypeReplace,
-		Arch:                           "amd64",
+		PullSecretFile:                 "",
+		ReleaseImage:                   "",
+		ReleaseStream:                  "",
```

Comment:
let's drop InfrastructureJSON unless there's known use case that is already strongly asking for this